src/x509.c

```diff
@@ -5871,8 +5877,11 @@ static int X509PrintSignature_ex(WOLFSSL_BIO* bio, byte* sig,
         return ret;
     }
 
+#if defined(WOLFSSL_QT)
```

Comment:
would like the print out formating to be the same for every build. Please make the offset just default to 5 and put a comment by it that the value 5 is required for QT test cases, that way if testing the expected output we do not need multiple expected results.